src/components/Nav/Menu/MenuList.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import MenuItem from './MenuItem';
+import './MenuList.scss';
+
+export class MenuList extends Component {
+  render() {
+    const { categories } = this.props;
+    return (
+      <div className="menu">
+        <ul className="menuList">
+          {categories.map(category => (
+            <MenuItem
+              key={category.id}
+              name={category.name}
+              list={category.list}
+            />
+          ))}
```

Comment:
```suggestion
          {categories.map(category => {
            const {id, name, list} = category;
            return (
            <MenuItem
              key={id}
              name={name}
              list={list}
            />
          )
          })}
```
구조분해할당해서 사용할 수 있겠네요!